modules/encryption/enc_md5.cpp

```diff
@@ -201,7 +201,11 @@ class MD5Context : public Encryption::Context
 		state[3] += d;
 
 		/* Zeroize sensitive information. */
+#ifdef _MSC_VER
```

Comment:
```suggestion
#ifdef _WIN32
```

Reply:
@SadieCat, fix it.
https://github.com/anope/anope/pull/295/commits/4fb0a3908ddd4305f06db76d8e60330a92e4a28e